src/utils.js

```diff
@@ -283,7 +288,39 @@ const recapAlertButton = (court, pacerCaseId, isActive) => {
   return anchor;
 };
 
+// Creates an anchor element to autofill the Docket Query form
+const recapAddLatestFilingButton = (result) =>{
+  
+  let date = result.date_last_filing
```

Comment:
When we don't have a docket or this field is null, does everything work properly?

Reply:
If We don't have a docket, the extension doesn't insert the button. but It is not working properly if this field is null. 

What should We do in such a case? avoid inserting the button or insert a button that only unchecks "Parties and counsel"?

let me know what you think.

Reply:
I hadn't mentioned it, but the ideal version of this button checks if we have party/counsel info and only unchecks if we do. That's hard, but at least in this case if we don't have the docket, we can just not show the button, and that's pretty close.

Reply:
Thanks for the explanation. I will make sure We don't show the button if we find one of these cases and I'll include tests to avoid regressions.